src/MiniCover/HitServices/Hit.cs

```diff
@@ -1,54 +1,29 @@
-﻿
-namespace MiniCover.Reports
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace MiniCover.HitServices
 {
-    internal abstract class Hit
+    [Serializable]
+    public class Hit
```

Comment:
I think it would be better to have only one Hit class that has a property like "TestMethods" that is nullable.
So, inheritance limits you in long term, composition would be better here.

Reply:
Totally agree on that, I was going to do that but was late ;p